### ISSUE TITLE
Unable to run

### ISSUE THREAD

**Issue by Wo0fle** [qqwref/sq1optim#2]
Just downloaded the latest version of sq1optim and it wont even open, this just comes up
![whyyyy](https://github.com/user-attachments/assets/140447f2-ea01-4be2-ac18-f2e8416838b6)
Only thing i did was download it, extract it, and run sq1opt.exe, and that pops up. CMD command wont work either.

**Comment by qqwref** [comment]
Yeah, I tried to make an exe that would work on my Win11 machine but couldn't, sorry. If you install WSL and then install g++ and mingw-w64 in the command prompt there, you should be able to run the .exe file inside WSL.
